css/selectors/adjacent_sibling.json

```
@@ -25,6 +25,7 @@
             "ie": {
               "version_added": "7",
               "notes": [
+                "Internet Explorer 7/8/9 supports <code>+~&gt;</code> combinators only in standards mode.",
```

style: this proposes conforming to our style elsewhere for indicating applicable versions for a note and avoids mentioning other features that aren't directly applicable (which can lead to maintenance hurdles, should the data change later)

```suggestion
                "Before Internet Explorer 10, the combinator only works in standards mode.",
```